scripts/gha/test_lab.py

```diff
@@ -295,7 +295,7 @@ def _gcloud_command(self):
     if self.platform == _ANDROID:
       cmd = ["gcloud", "firebase", "test", "android", "run"]
     elif self.platform == _IOS:
-      cmd = ["gcloud", "beta", "firebase", "test", "ios", "run"]
+      cmd = ["gcloud", "--quiet", "beta", "firebase", "test", "ios", "run"]
```

Comment:
Should the quiet flag be added to the Android command as well?

Reply:
There is no interactive prompts for android run. But probably yes.

Reply:
Change the components installation into yml.